### PR TITLE
remove ad label from ad templates

### PR DESCRIPTION
**Pull request by dskamiotis**
## What does this change?

This PR removes the insertion of ad Labels being rendered as part of the Native ad. 
We are removing it from source as we will now be rendering the ad label via the ad slots in commercial and DCR.

see related PR's : [commercial](https://github.com/guardian/commercial/pull/1484) and [DCR](https://github.com/guardian/dotcom-rendering/pull/12034)

## How to test
Checking out this branch and pnpm run dev and check: http://localhost:7777/csr/fabric

This was also tested with GAM line item:
https://admanager.google.com/59666047#delivery/line_item/detail/line_item_id=6756767022&li_tab=settings&sort_by=Status&sort_asc=false
You can add `?adtest=fluid-adlabel-hide` at the end of the URL

Creating a Native using the latest HTML and CSS resulting from local development and adding it to styles GAM Native Style [here](https://admanager.google.com/59666047#creatives/native/native_style/detail/style_id=869240&tab=ad_settings)

### **Important note**: Visual regression test failing:
`Expected an image 360px by 275px, received 360px by 251px. 14785 pixels (ratio 0.15 of all image pixels) are different.`
- This failure indicates the successful removal of the ad label.
- The "expected" image is dynamically generated by the Playwright test.
- This generated image is stored in the playwright/reference-images directory and reflects the current state of the source code. Result [here](https://guardian.github.io/commercial-templates/) 
- To update the reference image and align it with the recent changes, we need to merge the current branch into the main branch. This action will ensure the "expected" image is updated and future tests will be validated against the latest changes.

## How can we measure success?
We should hopefully not see any ad labels at the top of the add.

## Images

| Before. | After |
|-------| -------| 
|Screenshot 2024-07-31 at 15 02 10|Screenshot 2024-07-31 at 15 00 22|

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [x] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [x] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: legacy/lib/html.js
Comment (emma-imber) [inline legacy/lib/html.js:1]: Updating the legacy folders will have no impact on the templates in GAM. Only Svelte templates are automatically updated when we merge a PR in this repository. Any legacy templates will have to be manually updated in GAM. It might be sensible to mob/pair on this just before the commercial code change is released

File: legacy/src/fabric-custom/web/index.html
Comment (emma-imber) [inline legacy/src/fabric-custom/web/index.html:1]: No changes needed here - Fabric Custom has been migrated to Svelte

File: legacy/src/fabric/web/index.html
Comment (emma-imber) [inline legacy/src/fabric/web/index.html:1]: Fabric has been migrated to Svelte

Comment (dskamiotis) [reply legacy/src/fabric/web/index.html:1]: thanks @emma-imber, great spot

File: src/templates/csr/fabric-xl/index.svelte
Comment (emma-imber) [inline src/templates/csr/fabric-xl/index.svelte:8]: Just a heads up, no action required:

This variable will still exist in the native format as this codebase just updates the native styles. Updating the format is still something that needs to be done manually, but we should wait until after this PR is merged. I don't think there's any issues keeping the format as is for now - just some tidying up for later down the line.

**Review by emma-imber — APPROVED** [review]
Nice work!

File: src/templates/csr/fabric-xl/index.svelte
Comment (dskamiotis) [reply src/templates/csr/fabric-xl/index.svelte:8]: Great, ill keep an eye on the deploy 🙏

**Pull request merged**
